dsgrid/config/project_config.py

```diff
@@ -198,6 +198,7 @@ def check_version(cls, version):
 
     @validator("model_sector")
     def check_model_sector(cls, model_sector, values):
+        # TODO: how does change to "modeled" get set here? is this validator still relevant?
         if "dataset_type" in values:
             if not values["dataset_type"] == InputDatasetType.SECTOR_MODEL:
                 raise ValueError("model_sector is only required if dataset_type is 'sector_model'")
```

Comment:
I think it could still be reasonable to ask for an optional "sector" tag for "modeled" data sets.

Of course, historical and benchmark data sets can also have sectors, e.g., "electricity," "buildings," "all" ... Maybe it should just be dataset_sector and be an optional metadata tag? Does that make sense? 

Maybe it doesn't make sense here in the project config but makes more sense in the dataset config, because it's metadata and not necessarily something we want to enforce a strict match on between project and dataset?